eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -181,18 +195,33 @@ public function prepend(ContainerBuilder $container)
     private function getMainConfigParser()
     {
         if ($this->mainConfigParser === null) {
-            foreach ($this->configParsers as $parser) {
+            foreach ($this->siteAccessConfigParsers as $parser) {
                 if ($parser instanceof SuggestionCollectorAwareInterface) {
                     $parser->setSuggestionCollector($this->suggestionCollector);
                 }
             }
 
-            $this->mainConfigParser = new ConfigParser($this->configParsers);
+            $this->mainConfigParser = new ConfigParser($this->siteAccessConfigParsers);
         }
 
         return $this->mainConfigParser;
     }
 
+    private function getMainRepositoryConfigParser(): RepositoryConfigParserInterface
+    {
+        if ($this->mainRepositoryConfigParser === null) {
```

Comment:
To allow non-nullable typed property to work once the code gets updated to PHP 7.4+.
```suggestion
        if (!isset($this->mainRepositoryConfigParser)) {
```

Reply:
Fixed in https://github.com/ezsystems/ezplatform-kernel/pull/218/commits/69ff21ef0a40cbf323d7ef379f1bcaaffdca4f64